src/main/java/com/minecolonies/blocks/BlockHutField.java

```diff
@@ -0,0 +1,232 @@
+     */
+    private static final double BOTTOM_COLLISION = 0.0;
+
+    /**
+     * Start of the collision box at x and z.
+     */
+    private static final double START_COLLISION = 0.1;
+
+    /**
+     * End of the collision box.
+     */
+    private static final double END_COLLISION = 0.9;
+
+    /**
+     * Height of the collision box.
+     */
+    private static final double HEIGHT_COLLISION = 2.5;
+
+    /**
+     * Registry name for this block.
+     */
+    private static final String REGISTRY_NAME = "blockHutField";
+
+    /**
+     * Constructor called on block placement.
+     */
+    BlockHutField()
+    {
+        super(Material.wood);
+        initBlock();
+    }
+
+    /**
+     * Method called by constructor.
+     * Sets basic details of the block.
+     */
+    private void initBlock()
+    {
+        setRegistryName(REGISTRY_NAME);
+        setUnlocalizedName(Constants.MOD_ID.toLowerCase() + "." + "blockHutField");
+        setCreativeTab(ModCreativeTabs.MINECOLONIES);
+        //Blast resistance for creepers etc. makes them explosion proof.
+        setResistance(RESISTANCE);
+        //Hardness of 10 takes a long time to mine to not loose progress.
+        setHardness(HARDNESS);
+        this.setDefaultState(this.blockState.getBaseState().withProperty(FACING, NORTH));
+        GameRegistry.registerBlock(this);
+        setBlockBounds((float) START_COLLISION, (float) BOTTOM_COLLISION, (float) START_COLLISION, (float) END_COLLISION, (float) HEIGHT_COLLISION, (float) END_COLLISION);
+    }
+
+    @Override
+    public int getRenderType()
+    {
+        return -1;
+    }
+
+    @Override
+    public IBlockState getStateFromMeta(int meta)
+    {
+        EnumFacing facing;
+        switch (getFront(meta))
+        {
+            case WEST:
+                facing = WEST;
+                break;
+            case EAST:
+                facing = EAST;
+                break;
+            case NORTH:
+                facing = NORTH;
+                break;
+            default:
+                facing = SOUTH;
+        }
+        return this.getDefaultState().withProperty(FACING, facing);
+    }
+
+    @Override
+    public int getMetaFromState(@NotNull IBlockState state)
+    {
+        return state.getValue(FACING).getIndex();
+    }
+
+    @Override
+    public boolean isFullCube()
+    {
+        return false;
+    }
+
+    @Override
+    public boolean isPassable(IBlockAccess worldIn, BlockPos pos)
+    {
+        return false;
+    }
+
+    @Override
+    public boolean isOpaqueCube()
+    {
+        return false;
+    }
+
+    @NotNull
+    @Override
+    @SideOnly(Side.CLIENT)
+    public EnumWorldBlockLayer getBlockLayer()
+    {
+        return EnumWorldBlockLayer.SOLID;
+    }
+
+    @Override
+    public boolean onBlockActivated(@NotNull World worldIn, @NotNull BlockPos pos, IBlockState state, @NotNull EntityPlayer playerIn, EnumFacing side, float hitX, float hitY, float hitZ)
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Split this 186 characters long line (which is greater than 180 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS00103)